source/partials/_navbar.html.erb

```diff
@@ -1,4 +1,4 @@
-<% page_nav = %w[about location speakers schedule sponsors organizers] %>
+<% page_nav = %w[about location presentations schedule sponsors organizers] %>
```

Comment:
That isn't previous events. that's the link to presentations for the event being viewed since the navigation bar is still central to the current event. I need to figure out how to change that.

The reason it's now "Presentations" is because I inverted the topology of the event. Instead of listing speakers, I'm listing the presentations they give in order to support a "presentation has many speakers" relationship in each event.